pkgs/by-name/qs/qsv/package.nix

```
@@ -0,0 +1,138 @@
+    "--skip test_fetch::fetch_simple_url_template"
+    "--skip test_fetch::fetch_user_agent"
+    "--skip test_fetch::fetchpost_custom_user_agent"
+    "--skip test_fetch::fetchpost_literalurl_test"
+    "--skip test_fetch::fetchpost_simple_diskcache"
+    "--skip test_fetch::fetchpost_simple_test"
+    "--skip test_foreach::foreach_multiple_commands_with_shell_script"
+    "--skip test_geocode::geocode_countryinfo"
+    "--skip test_geocode::geocode_countryinfo_formatstr"
+    "--skip test_geocode::geocode_countryinfo_formatstr_pretty_json"
+    "--skip test_geocode::geocode_countryinfonow"
+    "--skip test_geocode::geocode_countryinfonow_formatstr"
+    "--skip test_geocode::geocode_countryinfonow_formatstr_pretty_json"
+    "--skip test_geocode::geocode_reverse"
+    "--skip test_geocode::geocode_reverse_dyncols_fmt"
+    "--skip test_geocode::geocode_reverse_fmtstring"
+    "--skip test_geocode::geocode_reverse_fmtstring_intl"
+    "--skip test_geocode::geocode_reverse_fmtstring_intl_dynfmt"
+    "--skip test_geocode::geocode_reverse_fmtstring_intl_invalid_dynfmt"
+    "--skip test_geocode::geocode_reversenow"
+    "--skip test_geocode::geocode_suggest"
+    "--skip test_geocode::geocode_suggest_dyncols_fmt"
+    "--skip test_geocode::geocode_suggest_dynfmt"
+    "--skip test_geocode::geocode_suggest_filter_country_admin1"
+    "--skip test_geocode::geocode_suggest_fmt"
+    "--skip test_geocode::geocode_suggest_fmt_cityrecord"
+    "--skip test_geocode::geocode_suggest_fmt_json"
+    "--skip test_geocode::geocode_suggest_intl"
+    "--skip test_geocode::geocode_suggest_intl_admin1_filter_country_inferencing"
+    "--skip test_geocode::geocode_suggest_intl_country_filter"
+    "--skip test_geocode::geocode_suggest_intl_multi_country_filter"
+    "--skip test_geocode::geocode_suggest_invalid"
+    "--skip test_geocode::geocode_suggest_invalid_dynfmt"
+    "--skip test_geocode::geocode_suggest_pretty_json"
+    "--skip test_geocode::geocode_suggestnow"
+    "--skip test_geocode::geocode_suggestnow_default"
+    "--skip test_geocode::geocode_suggestnow_formatstr_dyncols"
+    "--skip test_luau::luau_register_lookup_table"
+    "--skip test_luau::luau_register_lookup_table_on_dathere_url"
+    "--skip test_luau::luau_register_lookup_table_on_url"
+    "--skip test_sample::sample_seed_url"
+    "--skip test_snappy::snappy_decompress_url"
+    "--skip test_sniff::sniff_justmime_remote"
+    "--skip test_sniff::sniff_url_notcsv"
+    "--skip test_sniff::sniff_url_snappy"
+    "--skip test_sniff::sniff_url_snappy_noinfer"
+    "--skip test_to::to_parquet_dir"
+    "--skip test_validate::validate_adur_public_toilets_dataset_with_json_schema_url"
+  ];
+
+  env = { ZSTD_SYS_USE_PKG_CONFIG = true; };
+
+  meta = {
+    description = "CSVs sliced, diced & analyzed";
+    homepage = "https://github.com/jqnatividad/qsv";
+    changelog = "https://github.com/jqnatividad/qsv/blob/${src.rev}/CHANGELOG.md";
+    license = lib.licenses.mit;
```

See https://github.com/jqnatividad/qsv#license, dual-licensed.
```suggestion
    license = [
      lib.licenses.mit
      # or
      lib.licenses.unlicense
    ];
```